soda/core/soda/execution/check/check.py

```diff
@@ -110,6 +110,12 @@ def __init__(
         self.partition: Partition = partition
         self.column: Column = column
         self.metrics: dict[str, Metric] = {}
+        self.attributes = {}
+        self.soda_attributes = (
```

Comment:
We don't need soda_attributes  - everything is part of `atributes` - perhaps I wasnt' clear in the feature spec.

Reply:
removed, I was looking at outdated yaml spec 🤦